static/app/components/replays/walker/chevronDividedList.tsx

```diff
@@ -36,9 +36,9 @@ const List = styled('ul')<{cols: number}>`
   display: grid;
   gap: ${space(1)};
   grid-template-columns: ${p =>
-    `minmax(auto, max-content) repeat(${
-      (p.cols - 2) * 2 + 1
-    }, max-content) minmax(auto, max-content)`};
+    p.cols <= 3
+      ? `minmax(auto, max-content) max-content minmax(auto, max-content) max-content minmax(auto, max-content)`
+      : `minmax(auto, max-content) repeat(3, max-content) minmax(auto, max-content)`};
```

Comment:
in the <=3 case we're defining 5 columns: $content divider $content divider $content and we will use either 1, 3, or all of those columns to render

In the previous case, we are more specific, it's always 5 columns (3 content, 2 dividers), and the middle section isn't going to truncate.
This change means the component only works with up to 3 content columns max now.